examples/recipes/alex_line_detect.py

```diff
@@ -1,18 +1,18 @@
-from pitop import AlexRobot
+from pitop import Pitop, alex_configuration
 from pitop.processing.algorithms.line_detect import process_frame_for_line
 
 from signal import pause
 
 
-# Set up robot
-alex = AlexRobot()
+# Load Alex configuration into a Pitop object
+alex = Pitop.from_dict(alex_configuration)
```

Comment:
I would suggest that `from_config` is more appropriate here. It may be a dict, but it's a specifically organised dict that represents a pi-top's configuration.